pages/api/periods/ow-committees.ts

```diff
@@ -66,17 +67,14 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
             (group: { name_short: string }) =>
               !excludedCommitteeNames.includes(group.name_short) // Exclude committees by name_short
           )
-          .map(
-            (group: {
-              name_short: string;
-              name_long: string;
-              email?: string;
-            }) => ({
-              name_short: group.name_short,
-              name_long: group.name_long,
-              email: group.email || "No email provided",
-            })
-          )
+          .map((group: owCommitteeType) => ({
+            name_short: group.name_short,
+            name_long: group.name_long,
+            email: group.email,
+            description_short: group.description_short,
```

Comment:
Er nok best å kun bruke application_description som beskrivelse, da det er den som er rettet mot søkere

Reply:
Tror ikke de blir oppdatert. applicaation_description til velkom inneholder "Vi søker ny leder og nestleder til fadderukene 2020!"

Reply:
> Tror ikke de blir oppdatert. `application_description` til velkom inneholder "Vi søker ny leder og nestleder til fadderukene 2020!"

Da får heller de det gjelder oppdatere beskrivelsen på OW. Er i hvertfall den beskrivelsen som er mest skreddersydd opp mot søkerene

Reply:
Kan være enig i det. Må få sagt ifra så alle får med seg det da...

Flere utdaterte beskrivelser:

![image](https://github.com/user-attachments/assets/0370749d-62a4-44e0-ad0d-558f8bd15d1d)

![image](https://github.com/user-attachments/assets/9d9f0be3-4d7c-4b06-9db2-98d094fcbed4)

Reply:
Litt derfor jeg tenkte det var lurt å kun vise relevante komiteer for opptaket. Da Jubkom kun har opptak hvert 5. år, og Ekskom har opptak på våren